packages/client/tests/functional/count/_matrix.ts

```diff
@@ -0,0 +1,30 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'sqlite',
+      id: 'Int @id @default(autoincrement())',
+      foreignKeyId: 'Int?',
+      providerFeatures: '',
```

Comment:
I don't think you use `providerFeatures` in the schema, should we remove them from the matrix?